README.md

```diff
@@ -148,11 +148,11 @@ to create the project in the Actions Console, enter your Assistant settings.
 
     * If running locally:
 
-          node smart-home-provider-cloud.js isLocal
+          node cloud/smart-home-provider-cloud.js isLocal
```

Comment:
This should be `npm start`, to be consistent with what it stated in an earlier section.